sms_wsj/kaldi/get_kaldi_wer.py

```diff
@@ -199,8 +199,12 @@ def default():
         data_type = 'sms_enh'
 
     # only used with audio_dir
-    id_to_file_name = '{}_0.wav'
+    id_to_file_name = '{id}_{spk}.wav'
+    # id_to_file_name = '{}_{}.wav' is another possible default, but only
+    # if the first {} represents the example id and the second the speaker id
     target_speaker = [0, 1]
+    if isinstance(target_speaker, (list, tuple)) and len(target_speaker) > 1:
+        assert id_to_file_name.count('{}') == 2, id_to_file_name
```

Comment:
This does no longer work. 
Moving this to `create_data_dir_from_audio_dir` may simplify the test:
```python
if isinstance(target_speaker, (list, tuple)) and len(target_speaker) > 1:
    assert id_to_file_name_fn('example_id', 0) != id_to_file_name_fn('example_id', 1), (...)
```